src/test/java/com/artipie/rpm/CliTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm;
+
+import org.apache.commons.cli.ParseException;
+import org.hamcrest.MatcherAssert;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test case for {@link Cli}.
+ *
+ * @since 0.1
+ */
+class CliTest {
+
+    @Test
+    void testWrongArgumentCount() {
+        try {
+            Cli.main(new String[]{"a"});
+        } catch (final ParseException exception) {
+            MatcherAssert.assertThat(
+                String.format("Exception occurred: %s", exception.getMessage()),
```

Comment:
@ammaratef45 please assert exception message content.

Reply:
@Vatavuk well, I'm thinking how to put it while avoiding replication of the string,
I might create a custom exception with the message as a static filed inside it

Reply:
@ammaratef45 I don't see any problem in duplication. Its better to match an explicit string that doesn't depend on a production code.